### ISSUE TITLE
Publish new version and run npm release

### ISSUE THREAD

**Issue by scottgruber** [1000ch/gulp-image#137]
When trying to 

`npm install gulp-image --save-dev`

I get following warning.

```
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
```

When will the latest dependency updates in package.json show up in the npm repository? Is it related to the version number released?

Thanks for your help.

**Comment by scottgruber** [comment]
HI, 

Could you please issue a new release and update the npm repository via `npm run release`. 

Recent changes in this repo haven't propagated to https://www.npmjs.com/package/gulp-image and are causing install warnings and errors. 

Much thanks.

**Comment by 1000ch** [comment]
Published [v6.3.0](https://github.com/1000ch/gulp-image/releases/tag/v6.3.0).
